tests/test_routes.py

```diff
@@ -0,0 +1,58 @@
+# from flask.wrappers import Response
+
+# ^ huh ?? why was this in my hello books code ? 
+
+def test_get_planet_with_no_records(client):
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    assert response.status_code == 200
+    assert response_body == []
+
+def test_get_one_planet(client, two_planets):
+    response = client.get("/planets/1")
+    response_body = response.get_json()
+
+    assert response.status_code == 200
+    assert response_body == {
+        "id" : 1,
+        "name" : "venus",
+        "description" : "hottie w a body",
+        "moons" : 0
+    }
+
+def test_get_one_planet_with_no_records(client):
+    response = client.get("/planets/1")
+    response_body = response.get_json()
+
+    assert response.status_code == 404
+    assert response_body == "Error: Planet #1 not found"
+
+def test_get_all_planets(client, two_planets):
+    response = client.get("/planets")
+    response_body = response.get_json()
+
+    assert response.status_code == 200
+    assert response_body == [
```

Comment:
Without an explicit ordering in your GET route, it's a _little_ dangerous to require these to come back in this order. It would be a little safer to check that the two dictionaries are in the response body, without requiring this order.